src/test/java/com/wikia/webdriver/common/dataprovider/ads/GermanAdsDataProvider.java

```diff
@@ -7,7 +7,7 @@
  */
 public class GermanAdsDataProvider {
 
-  public static Object[][] IVW2_TEST_DATA = new Object[][]{
+  public static final Object[][] IVW2_TEST_DATA = new Object[][]{
       {"de.wikia", "Wikia", "RC_WIKIA_HOME"},
       {"de.wikia", "Spezial:Kontakt", "RC_WIKIA_SVCE"},
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "RC_WIKIA_SVCE" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)